CHANGES.rst

```diff
@@ -4,7 +4,12 @@ Changelog
 3.3.1 (unreleased)
 ------------------
 
+- Shifting buildout's .gitignore and buildout.cfg template to addon
```

Comment:
this line is not useful in the changelog, as it is a fix of the change you already mentioned.
Maybe better change the existing entry, to make it clear where this happend:

> Adding plone-compile-resources script in the buildout and extending .gitignore file